logsrvd/logsrvd.c

```diff
@@ -1172,16 +1279,23 @@ connection_closure_alloc(int sock)
     if (closure->commit_ev == NULL)
 	goto bad;
 
-    closure->read_ev = sudo_ev_alloc(sock, SUDO_EV_READ|SUDO_EV_PERSIST,
+    closure->read_ev = sudo_ev_alloc(sock, SUDO_EV_READ|SUDO_EV_PERSIST|SUDO_EV_TIMEOUT,
 	client_msg_cb, closure);
     if (closure->read_ev == NULL)
 	goto bad;
 
-    closure->write_ev = sudo_ev_alloc(sock, SUDO_EV_WRITE|SUDO_EV_PERSIST,
+    closure->write_ev = sudo_ev_alloc(sock, SUDO_EV_WRITE|SUDO_EV_PERSIST|SUDO_EV_TIMEOUT,
```

Comment:
Same here, no need for SUDO_EV_TIMEOUT

Reply:
fixed and amended to the corresponding commit